server/tests/events_ml_parser_test.py

```diff
@@ -187,3 +187,41 @@ def test_search_contacts_case_insensitive(self):
         self.assertEqual(
             search_existing_contacts(search_contact)["_id"], str(contact_ids[0])
         )
+
+    def test_location_saved_to_db(self):
+        locations_service = get_resource_service("locations")
+
+        self.location_data = {
+            "name": "Sokos Hotel Presidentti",
+            "address": {
+                "title": "Sokos Hotel Presidentti",
+                "city": "Helsinki",
+                "state": "Uusimaa",
+                "country": "Suomi",
+                "line": ["Eteläinen Rautatiekatu 4"],
+                "extra": {
+                    "sttlocationalias": "14068",
```

Comment:
would probably use that number to generate id so we don't rely on name when looking it up later, eg. generating something like `urn:stt:location:14068` as location id